lib/Backend/FlowGraph.cpp

```diff
@@ -22,18 +22,33 @@ IR::LabelInstr * FlowGraph::DeleteLeaveChainBlocks(IR::BranchInstr *leaveInstr,
 {
     // Cleanup Rest of the Leave chain
     IR::LabelInstr * leaveTarget = leaveInstr->GetTarget();
-    Assert(leaveTarget->GetNextRealInstr()->IsBranchInstr());
-    IR::BranchInstr *leaveChain = leaveTarget->GetNextRealInstr()->AsBranchInstr();
+    Assert(leaveTarget->GetNextBranchOrLabel()->IsBranchInstr());
```

Comment:
>GetNextBranchOrLabel [](start = 24, length = 20)

I'd rather add a new 'GetNextBranch' API instead of asserting that the return value of GetNextBranchOrLabel is a BranchInstr

Reply:
I changed it slightly to assert on the IR pattern we expect.